articles/data-explorer/ingest-data-event-hub.md

```diff
@@ -127,6 +127,9 @@ Now you connect to the event hub from Azure Data Explorer. When this connection
 
     > [!NOTE]
     > Select **My data includes routing info** to use dynamic routing, where your data includes the necessary routing information as seen in the [sample app](https://github.com/Azure-Samples/event-hubs-dotnet-ingest) comments. If both static and dynamic properties are set, the dynamic properties override the static ones. 
+    
+    > [!NOTE]
+    > Once the connection is set, it ingest data starting from events enqueued after its creation time.
```

Comment:
I would place both in one note with bullets.
reword:

"Only events enqueued after you create the data connection are ingested."